### PR TITLE
Fix for Error Message at Resources that have no breadcrumbs trait

### PR DESCRIPTION
**Pull request by behrang1981**
this fixes
Call to undefined method Vyuldashev\NovaPermission\Permission::breadcrumbs() {"userId":1,"exception":"[object] (Error(code: 0): Call to undefined method Vyuldashev\\NovaPermission\\Permission::breadcrumbs() at /Users/behrangalavi/PhpstormProjects/larabase/doc_root/vendor/chris-ware/nova-breadcrumbs/src/Http/Controllers/NovaBreadcrumbsController.php:53)

and removes empty breadcrumb container

### CODE REVIEW

**Comment by chris-ware** [comment]
@behrang1981 Looks like you've removed a couple of lines that aren't needed to be removed. Also, there isn't a need to assign the method exists variable, just run the function inline on the if, as it saves storing a variable that is only used once. Finally, please recompile for production.

I'm going to be looking to release a new update in the next few days, so will do all of the above if I get to it first.

**Comment by behrang1981** [comment]
sorry must have been late
i played too much around the lines 
now must be ok

File: src/Http/Controllers/NovaBreadcrumbsController.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ public function __invoke(Request $request)
         if ($pathParts->has(1)) {
             $this->resource = Nova::resourceForKey($pathParts->get(1));
 
-            if ($this->resource && ! $this->resource::breadcrumbs()) {
+            if ($this->resource && (! $this->resource::breadcrumbs() || ! method_exists($this->resource, 'breadcrumbs'))) {
```
Comment (mstaack) [inline src/Http/Controllers/NovaBreadcrumbsController.php:52]: shouldn't it be the second one? check method availability, then call it ?

**Review by mstaack — CHANGES_REQUESTED** [review]

File: src/Http/Controllers/NovaBreadcrumbsController.php (hunk 1 above)
Comment (behrang1981) [reply src/Http/Controllers/NovaBreadcrumbsController.php]: you are right, i had it changed but didnt push

**Comment by chris-ware** [comment]
I'm going to close this PR as I've implemented it a bit differently, using clearer early exist. I've also added an extra check to ensure that the resource is using the trait before it checks anything else.

**Pull request closed without merging**